tests-hivemq-platform-operator/gradle/docker.versions.toml

```
@@ -0,0 +1,7 @@
+[docker]
+# https://hub.docker.com/r/rancher/k3s/tags?page=1&name=v1.32
+k3s = { image = "rancher/k3s", tag = "v1.32.0-k3s1" }
+# https://hub.docker.com/_/nginx/tags?page=&page_size=&ordering=&name=1.27
+nginx = { image = "nginx", tag = "1.27.3" }
+# https://hub.docker.com/r/selenium/standalone-chrome/tags?name=4.
+selenium = { image = "selenium/standalone-chrome", tag = "4.27.0" }
```

Another test for Renovate maybe
```suggestion
selenium = { image = "selenium/standalone-chrome", tag = "4.28.0" }
```